shell/src/test/java/tachyon/shell/TFsShellUtilsTest.java

```diff
@@ -140,12 +145,12 @@ public static String resetLocalFileHierarchy(LocalTachyonCluster localTachyonClu
       throws IOException {
     /**
      * Generate such local structure /testWildCards
-     *                                ├── foo |
-     *                                        ├── foobar1
-     *                                        └── foobar2
-     *                                ├── bar |
-     *                                        └── foobar3
-     *                                └── foobar4
+                ├── foo |
+                        ├── foobar1
+                        └── foobar2
+                ├── bar |
+                        └── foobar3
+                └── foobar4
```

Comment:
Could you revert the formatting change (I think the lines are supposed to be coming down from `/testWildCards`?